website/frontend/src/components/header/Header.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import {NavLink} from 'react-router-dom';
+import './header.css'
+
+//import image
```

Comment:
can we remove this?

Reply:
This is a comment to organize my imports.